apps/native-ui-storybook/components/Alert/Alert.docs.mdx

```diff
@@ -68,40 +68,45 @@ const MyComponent = () => {
-| `onPressLink`       | `() => void`                                  | A callback function to be called when the link is clicked.         |         |
-| `onPressIconButton` | `() => void`                                  | A callback function to be called when the icon button is clicked.  |         |
-| `onClose`           | `() => void`                                  | A callback function to be called when the close button is clicked. |         |
+| Property            | Type                                   | Description                                                        | Default |
+| ------------------- | -------------------------------------- | ------------------------------------------------------------------ | ------- |
+| `colorScheme`       | `'cyan' \| 'green' \| 'gold' \| 'red'` | The colour scheme of the alert. Mapped to an icon.                 | 'info'  |
+| `title`             | `string`                               | The title of the alert.                                            |         |
+| `text`              | `string`                               | The text of the alert.                                             |         |
+| `link`              | `string`                               | The text of the link.                                              |         |
+| `onPressLink`       | `() => void`                           | A callback function to be called when the link is clicked.         |         |
+| `onPressIconButton` | `() => void`                           | A callback function to be called when the icon button is clicked.  |         |
+| `onClose`           | `() => void`                           | A callback function to be called when the close button is clicked. |         |
 
 For more information, see the [Gluestack Alert component documentation](https://gluestack.io/ui/docs/components/feedback/alert).
 
+## Variants
+
+The `Alert` component has four colour schemes: cyan, green, gold and red. Each colour scheme is mapped to an icon.
+
+<Canvas of={Stories.Variants} />
+
 ## Advanced Usage
 
 If you need to use the Alert component in a more advanced way, you can use the parts of the component directly.
 
 <Canvas>
   <NativeUIProvider>
-    <Alert colorScheme="info">
-      <AlertIcon as={InformationMediumContainedIcon} />
+    <Alert colorScheme="cyan">
+      <AlertIcon />
       <VStack flex={1} gap={4}>
         <AlertTitle>Information</AlertTitle>
         <AlertText>Unlock the power of knowledge with the following information.</AlertText>
-
         <AlertLink onPress={() => console.log('Link clicked')}>
           <AlertLinkText>Learn more</AlertLinkText>
-          <AlertLinkChevron as={ChevronRightSmallIcon} />
+          <AlertLinkChevron />
         </AlertLink>
       </VStack>
       <AlertIconButton onPress={() => console.log('Icon button clicked')}>
```

Comment:
I don't know if it's different in App UI, but in Web UI this is supposed to be a link rather than a button.

Reply:
There's no concept of links in RN, just pressable views so you don't have hrefs etc. So visually and conceptually it's a link but it can't be a link. Is that what you mean?